app/assets/javascripts/angular/controllers/my_books.js.erb

```diff
@@ -11,12 +11,10 @@ app.controller('MyBooksCtrl', ['$scope', '$http', '$location', '$auth', function
   $scope.form = {};
 
   $scope.form.create = function(){
-    var url = "<%= ENV['URL'] %>" || "http://localhost:3000";
-
     var data = {
       book: $scope.book
     };
-
+    data.book.condition /= 2;
```

Comment:
This is very stupid... In my database the "condition" field is set to be decimal ranged 0 to 5. I am using the angular built-in rating directive and couldn't manage to find a way to modify the value of each star. So now I am using 10 stars and divide them by 2 when submitting to the database.